selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/gatein/GateinLocator.java

```diff
@@ -1220,8 +1220,8 @@
   public static final String          ELEMENT_PORTAL_TEMPLATE_TAB                             =
                                                                   "//a[contains(text(),'Portal Templates')]";
 
-  public static final String          ELEMENT_SAVE_BUTTON                                     =
-                                                          "//button[contains(text(),'Save')]";
+  public static final SelenideElement        ELEMENT_SAVE_BUTTON                                     =
+                                                          $(byText("Save"));
```

Comment:
Don't modify an existed locator, it may impact other tests. if you can't reuse it, add a new one.